machine_types.go

```diff
@@ -813,8 +817,28 @@ type ContainerConfig struct {
 	// DependsOn can be used to define dependencies between containers. The container will only be
 	// started after all of its dependent conditions have been satisfied.
 	DependsOn []ContainerDependency `json:"depends_on,omitempty"`
+
+	// Healthchecks determine the health of your containers. Healthchecks can use HTTP, TCP or an Exec command.
+	Healthchecks []ContainerHealthcheck `json:"healthchecks,omitempty"`
+
+	// Set of mounts added to the container. These must reference a volume in the machine config via its name.
+	Mounts []ContainerMount `json:"mounts,omitempty"`
+}
+
+type ContainerMount struct {
+	// The name of the volume. Must exist in the volumes field in the machine configuration
+	Name string `json:"name"`
+	// The path to mount the volume within the container
+	Path string `json:"path"`
 }
 
+type StorageType string
```

Comment:
nit: move the `StorageType` definition closer to the struct where it's being used (i.e. `TempDirVolume`).